packages/react-core/src/components/ApplicationLauncher/ApplicationLauncherItem.tsx

```diff
@@ -3,11 +3,10 @@ import { css } from '@patternfly/react-styles';
 import styles from '@patternfly/react-styles/css/components/AppLauncher/app-launcher';
 import { DropdownItem, DropdownItemProps } from '../Dropdown';
 import { ApplicationLauncherContent } from './ApplicationLauncherContent';
-import { ApplicationLauncherContext } from './ApplicationLauncher';
+import { ApplicationLauncherContext } from './ApplicationLauncherContext';
+import { ApplicationLauncherItemContext } from './ApplicationLauncherItemContext';
 import StarIcon from '@patternfly/react-icons/dist/js/icons/star-icon';
 
-export const ApplicationLauncherItemContext = React.createContext({ isExternal: false, icon: null });
```

Comment:
Wouldn't removing an export be a breaking change? 

I believe it's fine to add ApplicationLauncherItemContext.ts , but perhaps remove this in v4?

Reply:
Good catch! I've reexported it in the `index.ts` file so this isn't breaking.